### DISCUSSION TITLE
[Vote ended on 2023-07-31] Change list of accepted licenses

### DISCUSSION THREAD

**Discussion by Andersson007** [ansible-community/community-topics#260] [Votes done]
Vote on merging https://github.com/ansible/ansible-documentation/pull/141

Related discussion https://github.com/ansible-community/community-topics/issues/252#issuecomment-1639862493

In gist, the PR replaces the link to Fedora license table (which does not exist any more) with a link to https://www.gnu.org/licenses/license-list.html#GPLCompatibleLicenses (thanks @leogallego for suggesting this).

Please vote with +1/-1

**Comment by Andersson007** [comment]
+1

**Comment by mariolenz** [comment]
+1

**Comment by leogallego** [comment]
+1

**Comment by gotmax23** [comment]
-1. I don't think we should stop using the Fedora list for license data, just not use it for GPL compatibility data. I'd require that the license is approved on the Fedora list and on the FSF GPL compatibility list. I'd prefer to get rid of the Debian list entirely, but we can keep it in addition if people feel strongly. The Fedora list provides a nice table, is more comprehensive, and reflects the opinion of Red Hat legal, which we need to follow.

**Comment by russoz** [comment]
+1

**Comment by felixfontein** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by acozine** [comment]
+1, but I also put a suggested change in [a review](https://github.com/ansible/ansible-documentation/pull/141#pullrequestreview-1555948255) that might bring @gotmax23 on board.
